Can not compile 0.8.0

Hi, I'm getting the following error when trying to compile v0.8.0.
I've made a new Cargo.toml and have only this for the deps:

```
[dependencies]
tiny-bip39 = "0.8.0"
```

```
$ cargo build
   Compiling tiny-bip39 v0.8.0
error[E0184]: the trait `Copy` may not be implemented for this type; the type has a destructor
   --> /home/parazyd/.cargo/registry/src/github.com-1ecc6299db9ec823/tiny-bip39-0.8.0/src/language.rs:116:24
    |
116 | #[derive(Debug, Clone, Copy, PartialEq, Zeroize)]
    |                        ^^^^ Copy not allowed on types with destructors
    |
    = note: this error originates in the derive macro `Copy` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0184`.
error: could not compile `tiny-bip39` due to previous error
```

Rust/Cargo version is `1.55.0-nightly`.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "tiny-bip39"
-version = "0.8.0"
+version = "0.8.1"
 authors = [
     "Stephen Oliver <steve@infincia.com>",
     "Maciej Hirsz <hello@maciej.codes>",

--- a/src/language.rs
+++ b/src/language.rs
@@ -114,7 +114,6 @@ mod lazy {
 /// [Mnemonic]: ./mnemonic/struct.Mnemonic.html
 /// [Seed]: ./seed/struct.Seed.html
 #[derive(Debug, Clone, Copy, PartialEq, Zeroize)]
-#[zeroize(drop)]
 pub enum Language {
     English,
     #[cfg(feature = "chinese-simplified")]

--- a/src/language.rs
+++ b/src/language.rs
@@ -1,7 +1,6 @@
 use crate::error::ErrorKind;
 use crate::util::{Bits, Bits11};
 use rustc_hash::FxHashMap;
-use zeroize::Zeroize;
 
 pub struct WordMap {
     inner: FxHashMap<&'static str, Bits11>,
@@ -113,7 +112,7 @@ mod lazy {
 ///
 /// [Mnemonic]: ./mnemonic/struct.Mnemonic.html
 /// [Seed]: ./seed/struct.Seed.html
-#[derive(Debug, Clone, Copy, PartialEq, Zeroize)]
+#[derive(Debug, Clone, Copy, PartialEq)]
 pub enum Language {
     English,
     #[cfg(feature = "chinese-simplified")]

--- a/src/mnemonic.rs
+++ b/src/mnemonic.rs
@@ -147,7 +147,7 @@ impl Mnemonic {
         // this also validates the checksum and phrase length before returning the entropy so we
         // can store it. We don't use the validate function here to avoid having a public API that
         // takes a phrase string and returns the entropy directly.
-        let entropy = Zeroizing::new( Mnemonic::phrase_to_entropy(&phrase, lang)?);
+        let entropy = Zeroizing::new(Mnemonic::phrase_to_entropy(&phrase, lang)?);
 
         let mnemonic = Mnemonic {
             phrase,

